### DISCUSSION TITLE
21.09.08 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#127] [회의록]
## 논의 내용 

- [x] 최근 4개 주소지 조회시 중복되는 주소 제거 되는지?(중복되지 않는 주소로 최대 4개) (데이지 -> BE)
- [x] products page에서 발견한 header로 인한 sideDrawer버그 공유 

### 디코 
- 햄버거 버튼 완료

**Comment by damilog** [comment]
마이페이지 - 주문내역에서 리뷰 작성하게 해야함

**Comment by damilog** [comment]
1. 마이페이지-주문내역-리뷰작성 - 제니
2. 마이페이지 주문내역 시간별조회, 페이지네이션 - 데이지
3. 판매자용 마이페이지 - 판매자 프로필 수정(사진, 자기소개)- 디코

**Comment by wjddnjswjd12** [comment]
상품 등록할때 photo 어떻게 상태 관리할지

**Reply by ha3158987** [reply]
판매자 프로필 사진도!!!  => 백엔드 분들이랑 이미지는 서버에 어떻게 관리할 지 수요일에 얘기해봐야할 것 같아요 :D
